spec/features/viewing_messages_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'orderly'
+feature 'Viewing messages on chitter home' do
+  scenario 'visiting the home page' do
+    visit('/')
+    expect(page).to have_content("Welcome to Chitter!")
+  end
+end
+
+feature 'Viewing messages in reverse chronological order' do
+  scenario 'displays peeps in reverse chronological order' do
+    visit('/')
+    click_link('Post a new peep')
+    fill_in('author', with: 'Victor')
+    fill_in('body', with: 'Hello world!')
+    fill_in('date', with: '2020-10-02 10:00:00')
+    click_button('post')
+
+    visit('/')
+    click_link('Post a new peep')
+    fill_in('author', with: 'John')
+    fill_in('body', with: 'Hello gang!')
+    fill_in('date', with: '2020-10-02 11:00:00')
+    click_button('post')
+
+    expect('John').to appear_before('Victor')
```

Comment:
Appreciation 
I like how you've incorporated this into your feature test.

Reply:
That was a cool gem ahah